src/Components/Feelings/Feelings.js

```diff
@@ -1,40 +1,23 @@
 import React from 'react';
 import './Feelings.css';
+import feelingsData from './feelingsData';
 import { Link } from 'react-router-dom';
 
-const Feelings = ({ setFeeling }) => {
+const Feelings = () => {
 
-  const selectFeeling = ({ target }) => {
-    setFeeling(target.classList[1]);
-  }
+  const feelingsLinks = feelingsData.map(feelingData => {
+    return feelingData.associatedFeelings.map(feel => {
+      return <Link to={`/what-should-you-do/${feel}`} key={feel} className='feeling'>{feel}</Link>
+    })
+  });
 
   return (
     <section className='feelings-page'>
       <h1>BEAM</h1>
       <h2 className='feelings-header'>How are you today?</h2>
-        <Link className='feelings-container' to='/what-should-you-do'>
-          <p className='feeling happy' onClick={selectFeeling}>Happy</p>
-          <p className='feeling happy' onClick={selectFeeling}>Content</p>
-          <p className='feeling happy' onClick={selectFeeling}>Optimistic</p>
-          <p className='feeling sad' onClick={selectFeeling}>Sad</p>
-          <p className='feeling sad' onClick={selectFeeling}>Lonely</p>
-          <p className='feeling sad' onClick={selectFeeling}>Depressed</p>
-          <p className='feeling surprised' onClick={selectFeeling}>Surpised</p>
-          <p className='feeling surprised' onClick={selectFeeling}>Confused</p>
-          <p className='feeling surprised' onClick={selectFeeling}>Excited</p>
-          <p className='feeling bad' onClick={selectFeeling}>Bad</p>
-          <p className='feeling bad' onClick={selectFeeling}>Stressed</p>
-          <p className='feeling bad' onClick={selectFeeling}>Tired</p>
-          <p className='feeling fearful' onClick={selectFeeling}>Fearful</p>
-          <p className='feeling fearful' onClick={selectFeeling}>Anxious</p>
-          <p className='feeling fearful' onClick={selectFeeling}>Insecure</p>
-          <p className='feeling angry' onClick={selectFeeling}>Angry</p>
-          <p className='feeling angry' onClick={selectFeeling}>Distant</p>
-          <p className='feeling angry' onClick={selectFeeling}>Frustrated</p>
-          <p className='feeling disguisted' onClick={selectFeeling}>Disguisted</p>
-          <p className='feeling disguisted' onClick={selectFeeling}>Disappointed</p>
-          <p className='feeling disguisted' onClick={selectFeeling}>Awful</p>
-        </Link>
```

Comment:
I feel a sense of relief seeing this all this red, this refactor is going to clean up that section so nicely. Thanks for taking this on.